addons/explosives/stringtable.xml

```diff
@@ -540,6 +540,20 @@
             <German>Entschärfungszeit für Nicht-Sprengstoffexperten erhöhen? Standard: Ja</German>
             <Czech>Zvýšit čas potřebný k dokončení akce pokud není specialista? Výchozí: Ano</Czech>
         </Key>
+        <Key ID="STR_ACE_Explosives_ExplodeOnDefuse_DisplayName">
+            <English>Explode on defusal?</English>
+            <Polish></Polish>
```

Comment:
Please remove the empty stringtable entries. It makes it harder for our translators to find untranslated entries.